packages/twenty-front/src/modules/workflow/search-variables/types/StepOutputSchema.ts

```diff
@@ -1,19 +1,30 @@
 import { InputSchemaPropertyType } from '@/workflow/types/InputSchema';
 
-type Leaf = {
+export type Leaf = {
   isLeaf: true;
   type?: InputSchemaPropertyType;
   icon?: string;
+  label?: string;
   value: any;
 };
 
-type Node = {
+export type Node = {
   isLeaf: false;
   icon?: string;
-  value: OutputSchema;
+  label?: string;
+  value: Record<string, Leaf | Node>;
 };
 
-export type OutputSchema = Record<string, Leaf | Node>;
+export type BaseOutputSchema = {
+  fields: Record<string, Leaf | Node>;
+};
+
+export type RecordOutputSchema = {
+  object: { nameSingular: string; fieldIdName: string } & Leaf;
+  fields: Record<string, Leaf | Node>;
```

Comment:
can you create a type for `Record<string, Leaf | Node>;`. `NodeValue` or `OutputFields` ?

Reply:
It will be BaseOutputSchema!